server/service/hosts.go

```diff
@@ -255,7 +255,11 @@ func (r getHostResponse) error() error { return r.Err }
 
 func getHostEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (interface{}, error) {
 	req := request.(*getHostRequest)
-	host, err := svc.GetHost(ctx, req.ID, false)
+	opts := fleet.HostDetailOptions{
+		IncludeCVEScores: false,
+		IncludePolicies:  true, // intentionally true to preserve existing behavior
```

Comment:
My understanding it that this endpoint is where it is most critical to preserve existing behavior. The free version does rely on this field.